internal/pkg/policy/self.go

```diff
@@ -218,6 +218,8 @@ func (m *selfMonitorT) updateState(ctx context.Context) (client.UnitState, error
 		return client.UnitStateStarting, nil
 	}
 
+	reportOutputHealth(ctx, m.bulker, m.log)
```

Comment:
Currently pinging remote outputs every 5s (default monitor interval) and writing out a doc to the output health data stream
We could change this to only write out a doc if the state changed.